src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -23,6 +23,8 @@ public ActivityAttribute ()
 #endif
 		public bool                   AllowTaskReparenting    {get; set;}
 		public bool                   AlwaysRetainTaskState   {get; set;}
+		public bool                   AutoRemoveFromRecents   {get; set;}
```

Comment:
The documentation doesn't say it, but this attribute was added in API-21, as per `grep`ping `$HOME/android-toolchain/sdk/paltforms/*/data/res/values/attrs_manifest.xml`.

It should thus be wrapped in `#if ANDROID_21`.